### ISSUE TITLE
Doesn't work or a fundamental user error

### ISSUE THREAD

**Issue by jlharri** [pistacchio/obsidian-enhanced-tables#5]
I installed the plugin and tried to get it to work but failed. 

I tried re-starting Obsidian. I copied your sample yaml code block. I tried selecting the entire table. I tried creating a small table. I tried going into Read mode. 

Your documentation doesn't indicate how to invoke the controls. There is no command to invoke. There are no error messages in the JS console. 

So, either it's user error or the plugin just doesn't work. 

Here's my sample: 

```yaml enhanced-tables

# hide-controls: true
hide-configuration: true
```

| this   | is        |
| ------ | --------- |
| dsdf   | fdsdfdfdf |
| a test | dkd       |

Here is my Obsidian version: 
About Obsidian
Version 1.5.8 (Installer 1.4.13)

Jerry

**Comment by emisjerry** [comment]
Your YAML codeblock doesn't seem to be correct. 
Maybe you can refer to this video (switch to English CC subtitles):
https://youtu.be/H84hDYwRWAM

**Comment by pistacchio** [comment]
Hi! What are you trying to achieve? The configuration you provided doesn't do a thing, it just hides the configuration, but it's just useless as it is, you didn't even specify any configuration for a column.

If you try the attached markdown file, for instance, it will hide the first column.
Can you confirm that it works?

Thanks

[Table Test.md](https://github.com/pistacchio/obsidian-enhanced-tables/files/14493667/Table.Test.md)

**Comment by jlharri** [comment]
Hello,
I tried the MD file you sent and it doesn't work either. I installed
Enhanced Tables, opened the file, and nothing happened. I re-started
Obsidian and it still does nothing. No messages on the JS console either.
Here is a screenshot of what it looks like with and without the table
selected:

[image: image.png]
[image: image.png]

I turned on the debugger to time each plugins load time. Here are all the
other plugins that loaded in this vault:
Total plugin setup: 1,070ms
obsidian-excalidraw-plugin: 339ms
vscode-editor: 252ms
dataview: 85ms
obsidian-markdown-formatting-assistant-plugin: 61ms
obsidian-kanban: 47ms
metadata-menu: 44ms
enhanced-tables: 39ms
nldates-obsidian: 28ms
omnisearch: 26ms
file-tree-alternative: 22ms
obsidian-charts: 22ms
theme-picker: 15ms
obsidian-style-settings: 15ms
workspaces-plus: 13ms
darlal-switcher-plus: 12ms
obsidian-plantuml: 11ms
pane-relief: 9ms
obsidian-outliner: 8ms
emoji-shortcodes: 8ms
tag-wrangler: 6ms
obsidian-remember-file-state: 4ms
maximise-active-pane-obsidian: 3ms
obsidian-show-file-path: 2ms
obsidian-excel-to-markdown-table: 2ms

Good luck. It seems like it might be a handy plugin to have.

Jerry
